homeassistant/components/zwave_js/light.py

```diff
@@ -426,3 +433,77 @@ def _calculate_color_values(self) -> None:
             self._rgbw_color = (red, green, blue, white)
             # Light supports rgbw, set color mode to rgbw
             self._color_mode = COLOR_MODE_RGBW
+
+
+class ZwaveBlackIsOffLight(ZwaveLight):
+    """
+    Representation of a Z-Wave light where setting the color to black turns it off.
+
+    Currently only supports lights with RGB, no color temperature, and no white channels.
+    """
+
+    def __init__(
+        self, config_entry: ConfigEntry, client: ZwaveClient, info: ZwaveDiscoveryInfo
+    ) -> None:
+        """Initialize the light."""
+        super().__init__(config_entry, client, info)
+
+        self._last_color: dict[str, int] | None = None
+        self._supported_color_modes.discard(COLOR_MODE_BRIGHTNESS)
+
+    @property
+    def brightness(self) -> int:
+        """Return the brightness of this light between 0..255."""
+        return 255
+
+    @property
+    def is_on(self) -> bool:
+        """Return true if device is on (brightness above 0)."""
+        return self.info.primary_value.value is not None and any(
```

Comment:
Should we return `None` if value is `None` as then we don't know what the state is, right? Nowadays the ToggleEntity supports returning `None` from `is_on` to mark unknown state.